osmaxx/profile/views.py

```diff
@@ -18,14 +18,15 @@
 class SendVerificationEmailMixin(object):
     RATE_LIMIT_SECONDS = 30
 
-    def _send_email_verification(self, profile):
-        if cache.get(profile.associated_user.id):
+    def _send_email_verification(self):
```

Comment:
I think keeping the profile as parameter isn't a good idea, but having this method as public method on `Profile` sounds good to me.